tests/unittests/test-ctextmetrics.cpp

```diff
@@ -0,0 +1,243 @@
+﻿/*! @file */
+/*
+	Copyright (C) 2021 Sakura Editor Organization
+
+	This software is provided 'as-is', without any express or implied
+	warranty. In no event will the authors be held liable for any damages
+	arising from the use of this software.
+
+	Permission is granted to anyone to use this software for any purpose,
+	including commercial applications, and to alter it and redistribute it
+	freely, subject to the following restrictions:
+
+		1. The origin of this software must not be misrepresented;
+		   you must not claim that you wrote the original software.
+		   If you use this software in a product, an acknowledgment
+		   in the product documentation would be appreciated but is
+		   not required.
+
+		2. Altered source versions must be plainly marked as such,
+		   and must not be misrepresented as being the original software.
+
+		3. This notice may not be removed or altered from any source
+		   distribution.
+*/
+
+#include <gtest/gtest.h>
+#include "view/CTextMetrics.h"
+#include <vector>
+#include <Windows.h>
+
+class CTextMetricsWithGDI : public testing::Test {
+protected:
+	CTextMetricsWithGDI() {
+		lf1.lfCharSet = DEFAULT_CHARSET;
+		std::wcscpy(lf1.lfFaceName, L"MS Gothic");
```

Comment:
```suggestion
		::wcscpy_s(lf1.lfFaceName, L"MS Gothic");
```
https://docs.microsoft.com/ja-jp/cpp/c-runtime-library/reference/strcpy-s-wcscpy-s-mbscpy-s?view=msvc-160

Reply:
次回からはセキュア版関数にしておきます…。